Tests/js/views/services/assets/ez-subitemboxviewservice-tests.js

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-subitemboxviewservice-tests', function (Y) {
+    var serviceTest,
+        Assert = Y.Assert, Mock = Y.Mock;
+
+    serviceTest = new Y.Test.Case({
+        name: "eZ Subitem Box View Service tests",
```

Comment:
please define 2 test cases, one for each public method you need to test (`getViewParameters` and `load`)